CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ option(SSG_SVG_IN_XCCDF_ENABLED "If enabled, the built XCCDFs will include the S
 option(SSG_SEPARATE_SCAP_FILES_ENABLED "If enabled, separate SCAP files (OVAL, XCCDF, CPE dict, ...) will be installed alongside the source data-streams" TRUE)
 option(SSG_ANSIBLE_PLAYBOOKS_ENABLED "If enabled, Ansible Playbooks for each profile will be built and installed." TRUE)
 option(SSG_BASH_SCRIPTS_ENABLED "If enabled, Bash remediation scripts for each profile will be built and installed." TRUE)
+option(SSG_ANSIBLE_TASKS_ENABLED "If enabled, Ansible Tasks for each rule will be installed." FALSE)
```

Comment:
In fact, these files are Ansible Playbooks, so using the term "tasks" is misleading.

Reply:
I understand that... but how should I call them since `SSG_ANSIBLE_PLAYBOOKS_ENABLED` is already being used?

maybe `SSG_ANSIBLE_INDIVIDUAL_PLAYBOOKS_ENABLED` or `SSG_ANSIBLE_PLAYBOOKS_PER_RULE_ENABLED`?

Reply:
I've changed to SSG_ANSIBLE_PLAYBOOKS_PER_RULE_ENABLED, see: https://github.com/ComplianceAsCode/content/pull/7039/commits/81f9051433bec735f0ce915290d465ba98401f86#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR51